server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessage.java

```diff
@@ -78,7 +80,7 @@ public String generateMessage(String parameterMessage, Mail originalMail) throws
         appendAddresses(builder, "To", message.getHeader(RFC2822Headers.TO));
         appendAddresses(builder, "CC", message.getHeader(RFC2822Headers.CC));
 
-        builder.append("  Size (in bytes): " + message.getSize())
+        builder.append("  Size: " + humanSize(message.getSize()))
```

Comment:
I don't think we need a static import for this one, SizeUtils.human(Readable)Size sounds fine.

Reply:
Why not ?

Reply:
we reserve static imports to DSL or very dependent cases. It's not the case here.

Reply:
done